http/src/main/resources/leo.conf

```diff
@@ -154,12 +152,21 @@ azure {
     enabled = ${?HAIL_BATCH_APP_ENABLED}
   }
 
+  coa-app-config {
+    instrumentation-enabled = ${?COA_INSTRUMENTATION_ENABLED}
+  }
+
+  wds-app-config {
+    instrumentation-enabled = ${?WDS_INSTRUMENTATION_ENABLED}
+    database-enabled  = ${?WDS_DATABASE_ENABLED}
```

Comment:
`database-enabled` is configurable by environment. As of this PR it will default to `false` in reference.conf.